dbms/src/Flash/Planner/plans/PhysicalWindow.cpp

```diff
@@ -92,6 +92,8 @@ void PhysicalWindow::finalize(const Names & parent_require)
 
     child->finalize(window_description.before_window->getRequiredColumns());
     FinalizeHelper::prependProjectInputIfNeed(window_description.before_window, child->getSampleBlock().columns());
+
+    window_description.fillArgColumnNumbers();
```

Comment:
Why not fill the arg column numbers during `transfrom` stage, since agg function fill the arg column in that stage, it looks weird to me if `window function` and `agg function` fill the arg column in different stage.

Reply:
ok